diff --git a/gitconfig b/gitconfig
index d0b5ee8..2b17743 100644
--- a/gitconfig
+++ b/gitconfig
@@ -1,6 +1,9 @@
 [user]
-    email = zl256@cornell.edu
+    email = zli@brex.com
     name = Zhijiang Li
+[credential "https://github.com"]
+    helper = "/opt/sandboxd/sbin/wsenv git-credentials"
+    useHttpPath = true
 [color]
     ui = auto
 [core]
diff --git a/zshrc b/zshrc
index 4555554..5097487 100644
--- a/zshrc
+++ b/zshrc
@@ -112,5 +112,8 @@ alias ll='ls -lhF --group-directories-first'
 alias la='ls -lhaF --group-directories-first'
 alias g='git'
 
+alias brexctl='b'
+alias myb="bazel run //foundation_platform/cli/cmd/brexctl --"
+
 # ubuntu open alias
 # alias open='xdg-open'
