diff --git a/gitconfig b/gitconfig
index 78febc1..d612da2 100644
--- a/gitconfig
+++ b/gitconfig
@@ -1,5 +1,8 @@
+[credential "https://github.com"]
+  helper = "/opt/sandboxd/sbin/wsenv git-credentials"
+  useHttpPath = true
 [user]
-       email = zl256@cornell.edu
+       email = zli@brex.com
        name = Zhijiang Li
 [color]
        ui = auto
diff --git a/zshrc b/zshrc
index 1f8c433..6f739d5 100644
--- a/zshrc
+++ b/zshrc
@@ -1,5 +1,5 @@
 # Path to your oh-my-zsh installation.
-export ZSH="/home/zli/.oh-my-zsh"
+export ZSH="/home/$USER/.oh-my-zsh"

 # Set name of the theme to load --- if set to "random", it will
 # load a random theme each time oh-my-zsh is loaded, in which case,
