diff --git a/gitconfig b/gitconfig
index 78febc1..cee5528 100644
--- a/gitconfig
+++ b/gitconfig
@@ -1,6 +1,9 @@
 [user]
-       email = zl256@cornell.edu
+       email = zli@brex.com
        name = Zhijiang Li
+[credential "https://github.com"]
+    helper = "/opt/sandboxd/sbin/wsenv git-credentials"
+    useHttpPath = true
 [color]
        ui = auto
 [core]
@@ -31,6 +34,3 @@
        process = git-lfs filter-process
        required = true

-# This next lines include Netlify's Git Credential Helper configuration in your Git configuration.
-[include]
-  path = /home/zli/.netlify/helper/git-config
diff --git a/zshrc b/zshrc
index 1f8c433..6f739d5 100644
--- a/zshrc
+++ b/zshrc
@@ -1,5 +1,5 @@
 # Path to your oh-my-zsh installation.
-export ZSH="/home/zli/.oh-my-zsh"
+export ZSH="/home/$USER/.oh-my-zsh"

 # Set name of the theme to load --- if set to "random", it will
 # load a random theme each time oh-my-zsh is loaded, in which case,
